app/controllers/admin/users_controller.rb

```diff
@@ -26,8 +42,15 @@ def load_user
     render "shared/not_found"
   end
 
-  def load_params
-    @page = params[:page]
-    @type = params[:type]
+  def params_user
+    params.require(:user).permit :name, :email,
+                                 :password, :password_confirmation,
+                                 :birthday, :company, :gradution,
+                                 :university, :address, :role, :gender
+  end
+  def message_exeption
+    @message = ProcessingDeleteUserCourseTasksAndUserCourseSubjectAndUserCourse.get_message
```

Comment:
`ProcessingDeleteUserCourseTasksAndUserCourseSubjectAndUserCourse.get_message` thế này thì dùng chung 1 mesage cho toàn bộ thread ah em thấy có sai sai ko